MVP Module/Tests/Feed UI/FeedUISnapshotTests.swift

```diff
@@ -51,6 +51,6 @@ class FeedUISnapshotTests: XCTestCase {
 
 private extension FeedViewController {
 	func display(errorMessage: String) {
-		fatalError("Must be implemented - follow the MVC solution as a guide")
+		errorView?.show(message: errorMessage)
```

Comment:
If at some point error displaying logic will get more complex other than just simply showing a message in `errorView` then this method would potentially break the test. To decouple the test I recommend using `FeedErrorViewModel` and `display` method of `FeedErrorView` protocol.
```suggestion
		display(FeedErrorViewModel(message: errorMessage))
```